.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: java
 jdk:
-- openjdk7
 - oraclejdk7
```

Comment:
I would agree.  Trouble is, since it has reached end-of-life, Oracle doesn't keep a copy publicly available, and the license doesn't allow folks to leave it on-line anywhere.  So, while people (TiVo included) still use JDK 1.6, we can't use it at Travis.  We still target 1.6 in the compilation options, though, so anything compiled will be able to run on a 1.6 JVM.  so far, when I've made manual builds, I have used a 1.6 compiler.